src/components/FeatureTrainingModal.tsx

```diff
@@ -397,14 +403,13 @@ function FeatureTrainingModal({
                             text={helpText}
                         />
                     )}
-                    <Button
-                        large
-                        success
-                        pressOnEnter
-                        onPress={closeAndConfirmModal}
+                    <FormAlertWithSubmitButton
+                        onSubmit={closeAndConfirmModal}
                         isLoading={shouldShowConfirmationLoader}
-                        text={confirmText}
+                        buttonText={confirmText}
+                        enabledWhenOffline={canConfirmWhileOffline}
                     />
+                    {!canConfirmWhileOffline && <OfflineIndicator addBottomSafeAreaPadding />}
```

Comment:
This is not rendered correctly on native

<img width="489" alt="Screenshot 2025-05-11 at 6 05 43 PM" src="https://github.com/user-attachments/assets/548b8479-9228-4f4c-9646-77190dfdbfa9" />